allennlp/models/coreference_resolution/coref.py

```diff
@@ -605,17 +815,23 @@ def _compute_coreference_scores(
 
         # Parameters
 
-        pairwise_embeddings : `torch.FloatTensor`, required.
-            Embedding representations of pairs of spans. Has shape
-            (batch_size, num_spans_to_keep, max_antecedents, encoding_dim)
-        top_span_mention_scores : `torch.FloatTensor`, required.
-            Mention scores for every span. Has shape
+        top_span_embeddings : `torch.FloatTensor`, required.
+            Embedding representations of the kept spans. Has shape
+            (batch_size, num_spans_to_keep, embedding_size)
+        top_antecedent_embeddings: `torch.FloatTensor`, required.
+            The embeddings of antecedents for each span candidate. Has shape
+            (batch_size, num_spans_to_keep, max_antecedents, embedding_size)
+        top_partial_antecedents_scores : `torch.FloatTensor`, required.
```

Comment:
Can you explain why this is partial, it's not clear right now what it is missing

Reply:
Explained in the doc.